lumigator/python/mzai/backend/backend/services/jobs.py

```diff
@@ -261,12 +263,16 @@ def create_job(
         # (request) and on the job type
         config_params = self._get_job_params(job_type, record, request)
 
+        loguru.logger.info(f"sending config_params...{config_params}")
+
         # load a config template and fill it up with config_params
-        if request.config_template is not None:
+        if request.config_template is not None and request.config_template != "":
```

Comment:
this failed if you input a config template that's an empty string, putting this here so it saves the next person time, will add a unit test around it as well.